### ISSUE TITLE
iPhone 7 and iPad 6 slow down and apps begin to hang and won’t open at all after a week of uptime

### ISSUE THREAD

**Issue by anthony6116** [checkra1n/BugTracker#2067]
I’m able to jailbreak my devices just fine but after around 5 days my phone will start getting slower and apps will stop opening and just hang/freeze nothing will work at all until i reboot losing my jailbreak i have to do this every week. I thought maybe it could be my tweaks so i deleted everything except Cydia Substrate, Filza, and Cocoatop thats it and it still happened so then i fully DFU restored my phone in iTunes and set it up as new then i rejailbroke not installing anything except cydia and it’s updates and the same thing happened again after a week later. This also happens on my iPad 6.

**Comment by Siguza** [comment]
If you get to a state again where apps don't open at all, could you connect via USB and see if SSH still works?
And if it does, could you run `spindump` as root? That should drop a log in `/tmp` that should tell us more.

**Comment by anthony6116** [comment]
Yes i’ll definitely try that when/if it happens again. I’m currently trying odysseyra1n right now but hasn’t been a week yet so i’ll see. If i get those logs how do i upload them?

**Comment by Siguza** [comment]
Just drag it into the comment field here on GitHub. Zip it if necessary.

**Comment by anthony6116** [comment]
> Just drag it into the comment field here on GitHub. Zip it if necessary.

Okay so it’s been around 8 days since i reboot/rejailbroke with checkra1n and my phone is starting to slow down like i cant have more than one app in the background and sometimes apps just crash with only one app open it will eventually not let me open any app soon until i reboot. I notice in cocoatop under the memory tab that the (kernel_task) starts out at around like 300mb or so when i reboot/rejailbreak but after a week of uptime it ends up over 900mb.

Should i send you the spindump log file now or wait until apps aren’t loading at all anymore? As of right now i can still open apps but only one so it’s starting to get bad again.
It’s the same it’s been doing i tried using odysseyra1n and same thing.
While my apps can still be opened for now i noticed the camera app opens but crashes when trying to take a picture also webpages will constantly keep refreshing.

**Comment by Siguza** [comment]
Hmm. If the actual issue is memory pressure, then a spindump isn't gonna help, really.

**Comment by anthony6116** [comment]
> Hmm. If the actual issue is memory pressure, then a spindump isn't gonna help, really.

So what can i do? 

Here it is anyway:
[spindump.txt](https://github.com/checkra1n/BugTracker/files/6468676/spindump.txt)




Also here is a screenshot of my cocoatop:

![98FBA680-493F-4326-81F9-CFA7AD335E31](https://user-images.githubusercontent.com/41974703/118033804-3b09c000-b337-11eb-8f22-9964c636f84e.png)

**Comment by anthony6116** [comment]
> Hmm. If the actual issue is memory pressure, then a spindump isn't gonna help, really.

I just don’t get why there’s some people on the same version and it works just fine but then there’s the select few that i’ve even seen on reddit that have this exact same problem and a lot of them are on iPhone 7. Maybe something to do with the A10 processor.

Okay now i’m seeing more and more people post about this issue on reddit

**Comment by theguywiththeproblem** [comment]
Hey, I got the same problem. I jailbroke with checkra1n and didn’t install the bootstrap and the issue still occurred. So it has nothing to do with tweaks or other installed software.

**Comment by anthony6116** [comment]
> Hey, I got the same problem. I jailbroke with checkra1n and didn’t install the bootstrap and the issue still occurred. So it has nothing to do with tweaks or other installed software.

Are you also on an iPhone 7?

**Comment by theguywiththeproblem** [comment]
Yep iPhone 7, 14.4.1. This happens on 14.3 too.

**Comment by anthony6116** [comment]
> Yep iPhone 7, 14.4.1. This happens on 14.3 too.

Ah yes all i see are A10 devices with this problem and since it’s the same on my iPad 6 (same A10 processor) it has to be something with these devices that are just not compatible in some way with this jailbreak

**Comment by anthony6116** [comment]
> Yep iPhone 7, 14.4.1. This happens on 14.3 too.

Also have you tried other jailbreaks like unc0ver for 14.3? On your iPhone 7 ?

**Comment by theguywiththeproblem** [comment]
When I was on 14.3, there wasn’t a semi-untethered jailbreak yet. I had to update because I deleted a system file. By the way, do you have a discord? I’ll send you an invite to the checkra1n server.

**Comment by anthony6116** [comment]
> When I was on 14.3, there wasn’t a semi-untethered jailbreak yet. I had to update because I deleted a system file. By the way, do you have a discord? I’ll send you an invite to the checkra1n server.

Ohh yeah i was already on the latest version when i decided to jailbreak and haven’t used any others on this iPhone. Yes i do have a discord

**Comment by theguywiththeproblem** [comment]
Alright, I’ll pm you on reddit about it.
